blurb/blurb.py

```diff
@@ -112,17 +111,37 @@
 
 def sanitize_section(section):
     """
-Cleans up a section string, making it viable as a directory name.
+    Clean up a section string, making it viable as a directory name.
+    """
+    return section.replace("/", "-").replace(" ", "_")
```

Comment:
To make things more explicit/readable, this could use a `dict` similar to the one used below for `unsanitize_section`.
If new sections that require sanitation are added in the future the function won't work, but it will serve as a reminder to update both `dict`s.

Reply:
Updated.